apps/token/src/routes/withdraw/eth-address-input.tsx

```diff
@@ -31,7 +31,7 @@ export const EthAddressInput = ({
   }, [connectedAddress, onChange, useConnectedWallet]);
 
   return (
-    <FormGroup label={t('To')} labelFor="ethAddressInput">
```

Comment:
chore (blocking): Why remove this? We'll lose the a11y benefit

Reply:
@mattrussell36 I've refactored the Formgroup to nest the child content inside the label tag. As I understand it, this implicit association works well with modern assistive tech, so we should get the same a11y benefit without the extra prop and faff.

Reply:
Oh right, aware you can nest fields. I hadn't noticed that form group now does this